source/parser.hera

```diff
@@ -44,7 +44,8 @@ Expression
 
 # https://262.ecma-international.org/#prod-Arguments
 Arguments
-  OpenParen ArgumentList? ( __ Comma )? __ CloseParen
+  ExplicitArguments ->
```

Comment:
If the rule is only a single expression it is automatically returned so you don't need to add

```
->
  return $1
```

Reply:
Oh, I didn't know that — thanks!